presto-product-tests/pom.xml

```diff
@@ -131,7 +131,9 @@
                     <groupId>org.gaul</groupId>
                     <artifactId>modernizer-maven-plugin</artifactId>
                     <configuration>
-                        <exclusionsFile>${project.basedir}/src/modernizer/exclusions.txt</exclusionsFile>
+                        <exclusions>
+                            <exclusion>com/google/inject/Inject</exclusion>
```

Comment:
Looks like 1.5.0 will remove this rule, so this should be removable entirely

(though this unfortunate, as we actually want to enforce usage of `javax.inject.Inject` everywhere...)